cirq/__init__.py

```diff
@@ -326,6 +328,7 @@
     SupportsQasm,
     SupportsQasmWithArgs,
     SupportsQasmWithArgsAndQubits,
+    SupportsQidShape,
```

Comment:
Should be two imports since there's two new protocols, right?

Reply:
I consider this a single protocol with two magic methods.  Should I split them into two protocols?

Reply:
Fixed.